provisioner/secret.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+var base = require('taskcluster-base');
+var assert = require('assert');
+var debug = require('debug')('aws-provisioner:Secret')
+var taskcluster = require('taskcluster-client');
+
+var Secret = base.Entity.configure({
+  version: 1,
+
+  // TODO: verify these are valid azure keys - heeeelp please Jonas
+  partitionKey: base.Entity.keys.StringKey('token'),
+  rowKey: base.Entity.keys.StringKey('provisionerId'),
```

Comment:
I would recommend:
```js
partitionKey: base.Entity.keys.StringKey('token'),
rowKey: base.Entity.ConstantKey('secret')
```

Note, I don't see any reason for `provisionerId` different provisioners should use different azure tables, I think that's safest :)